packages/frontend/src/contexts/AppContext/useBridges.ts

```diff
@@ -12,11 +13,12 @@ const useBridges = (sdk: Hop) => {
     })
   }, [sdk])
 
-  const queryParamBridge = useMemo(() => {
-    return bridges.find(bridge => bridge.getTokenSymbol() === queryParams.token)
-  }, [bridges, queryParams])
+  const queryParamBridge = useMemo(
+    () => findMatchingBridge(bridges, queryParams.token as string),
+    [bridges, queryParams]
+  )
 
-  const [selectedBridge, _setSelectedBridge] = useState<HopBridge>(queryParamBridge ?? bridges[0])
+  const [selectedBridge, _setSelectedBridge] = useState<HopBridge>(queryParamBridge ?? bridges[4])
```

Comment:
why is it defaulting to bridge index `4`? I think the order should be sorted and use `0` and the order of supported tokens and chains can also be customized via the environment variables `REACT_APP_ENABLED_TOKENS` and `REACT_APP_ENABLED_CHAINS`

Reply:
agreed!